src/electron/platform/android/device-focus-command-sender.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import axios from 'axios';
+
+export enum DeviceFocusCommand {
```

Comment:
I think this looks okay for now but I do wonder whether this should be named Visualization/VisualHelper commands, since I figure they'd be applicable to other scenarios.

Reply:
I hadn't thought of that, but it would seem a little odd since we hardcode "/FocusTracking/" into the URI that this class produces. I guess I assumed that other scenarios would have a different URI path. I wanted to avoid having the client send 2 commands (for example, one to hide visualizations and one to disable focus tracking), since 2 commands would complicate the state management if either one failed.

It's an easy refactor if/when we have additional scenarios to take into account. We already have 2 other classes that build a similar URI, so perhaps we'd pull all of those into a single location (or we might just decide that we ought to have a shared formatting string that gets used to build all URI's)